### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by welpaolo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,20 @@ An operator charm providing an integrator for connecting to S3 provides.
...
+#### Charmhub
+```shell
+juju deploy s3-integrator --channel edge
+```
+#### From source
+```shell
+git clone https://github.com/canonical/s3-integrator.git
+cd s3-integrator/
+sudo snap install charmcraft --classic
+charmcraft pack
```
Comment (MiaAltieri) [inline README.md:19]: this actually requires that LXD was configured, please add:
```
lxd init --auto
lxc network set lxdbr0 ipv6.address none 
```

**Review by MiaAltieri — CHANGES_REQUESTED** [review]
Awesome addition!! So happy its available now on charmhub. Only one change needed :)

File: README.md (hunk 1 above)
Comment (welpaolo) [reply README.md:21]: Done, nice catch :)

**Review by deusebio — APPROVED** [review]

**Review by MiaAltieri — APPROVED** [review]
wowowowowowow 🤩

**Pull request merged**